rollup.config.js

```diff
@@ -0,0 +1,23 @@
+import commonjs from 'rollup-plugin-commonjs';
+import builtins from 'rollup-plugin-node-builtins';
+
+export default {
+  input: 'index.js',
+  output: {
+    file: 'dist/index.umd.js',
+    format: 'umd',
+    name: 'devicePilot',
+    globals: {
```

Comment:
I think these also need to be `externals`: https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency